packages/@aws-cdk/cloudformation-diff/lib/format.ts

```diff
@@ -67,6 +68,89 @@ export function formatSecurityChanges(
   formatSecurityChangesWithBanner(formatter, templateDiff);
 }
 
+/**
+ * Renders stack drift information to the given stream
+ *
+ * @param stream The stream to write the formatted drift to
+ * @param driftResults The stack resource drifts from CloudFormation
+ * @param allStackResources A map of all stack resources
+ * @param verbose Whether to output more verbose text (include undrifted resources)
+ * @param logicalToPathMap A map from logical ID to construct path
+ */
+export function formatStackDriftChanges(
```

Comment:
I don't think this function should live in this package since it's not used in `@aws-cdk/cloudformation-diff` it at all. We can move this to `toolkit-lib`.  WDYT @rix0rrr ?

Reply:
Fair point. I guess this now lives with the other formatting code, but I do in general like scoping code to the smallest scope possible. Which means a private function in `toolkit-lib` is better than a shared function in `cloudformation-diff` (that's only intended to be used from `toolkit-lib`).